SmartDeviceLink_Example/Classes/AppDelegate.m

```diff
@@ -5,6 +5,7 @@
 #import "AppDelegate.h"
 
 #import "ProxyManager.h"
+#import "SDLLogMacros.h"
```

Comment:
What is the purpose importing the SDLLogMacros class in the AppDelegate if it is not being used?